### ISSUE TITLE
a11y issue in Flexible column layout: No way to pass land mark text  

### ISSUE THREAD

**Issue by lateefsofi** [SAP/ui5-webcomponents#2464]
**Describe the bug**
There is no way to pass the land mark text for Flexible Column Layout. Screen readers will always announce First Column region/Second column region. This text should be meaningful and related to the content which it represents.
**To reproduce**
1. Go to 'https://sap.github.io/ui5-webcomponents/playground/components/FlexibleColumnLayout/'
2. With JAWS running, Press INSERT+CNTL+R to inspect the landmarks on the page

**Isolated example**
https://sap.github.io/ui5-webcomponents/playground/components/FlexibleColumnLayout/

**Expected behavior**
Flexible Column layout should accept attributes which will set aria-labelledby for the regions.

**Screenshots**

**Context**
- UI5 Web Components version: 0.23.6
- OS/Platform: {...}
- Browser *(if relevant)*: {...}
- Other information: {...}

**Affected components** *(if known)*
Flexible Column layout

**Organization:** *(if applicable)*
SAP SuccessFactor

**Priority:** *(optional)* *(Low, Medium, High or Very High)*
Very High

![image](https://user-images.githubusercontent.com/18447407/98895212-95cb7780-24cc-11eb-9433-2913dd8bd486.png)
